src/components/NightMode/NightMode.tsx

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React, { useEffect } from 'react';
+import { FiMoon, FiSun } from 'react-icons/fi';
+
+import { ComponentPropsWithoutRef } from 'react';
+
+export type NightModeProps = ComponentPropsWithoutRef<'button'>;
+
+export function NightMode(props: NightModeProps) {
+  const { ...rest } = props;
+
+  const containerFlexClass = 'flex justify-center items-center';
+  const [theme, setTheme] = React.useState<'light' | 'dark'>('dark');
```

Comment:
Is there any reason to have setTheme here?

Reply:
It's just to know what icon display:

```ts
        {theme === 'dark' ? (
          <FiSun className="h-8 w-8" />
        ) : (
          <FiMoon className="h-8 w-8" />
        )}
```

Reply:
Let's sync in a call!